osf/models/metadata.py

```diff
@@ -22,12 +38,39 @@ class Meta:
     def serializer(self):
         return serializer_registry[self.schema._id]
 
+    def validate_user_entered_metadata(self):
```

Comment:
Dupe of above function? https://github.com/CenterForOpenScience/osf.io/pull/8710/files#diff-07f254bb1ed527862d14bc2f22915af5R18